firestore/firestore.rules

```diff
@@ -0,0 +1,27 @@
+rules_version = '2';
+service cloud.firestore {
+  match /databases/{database}/documents {
+
+    // Rules for the messages collection
+    match /messages/{messageId} {
```

Comment:
shouldn't this be a bit more nuanced.... e.g. /messages/{experimentID}/groupChat/{groupChatId}/{messageId} or something like that?

Reply:
You are right, I defined the current firestore layout by lazily translating the previous draft `AppState` with my limited knowledge of firestore which made it too monolithic in some places and overly nested in others. Now that the app completely relies on Firebase and that I understand it better, I will take some time to think about a better database structure and take this into account.